packages/slate-tools/tools/webpack/get-user-webpack-config.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable global-require, import/no-dynamic-require */
-const {getSlateConfig} = require('@shopify/slate-config');
+const config = require('../../slate-tools.config');
```

Comment:
This is loading the package schema and not the actual user config. Because of this, when trying to build a theme, it does not consider my config and therefore does not run my webpack extends.

Reply:
Really need to add tests to this package...

Reply:
This is hooked up correctly. What I forgot is to add `extends` to the schema.

Reply:
🤦‍♂️ my bad @t-kelly, you are correct.